code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1180,11 +1182,12 @@
 						H.glasses.add_blood(H)
 						H.update_inv_glasses()
 
-			if("chest")	//Easier to score a stun but lasts less time
-				if(H.stat == CONSCIOUS && I.force && prob(I.force + 10))
-					H.visible_message("<span class='danger'>[H] has been knocked down!</span>", \
-									"<span class='userdanger'>[H] has been knocked down!</span>")
-					H.apply_effect(5, WEAKEN, armor_block)
+			if("chest")	//Causes weakness and forces the target to drop their items
+				if(H.stat == CONSCIOUS && I.force && prob(min(I.force, 35)))
+					H.visible_message("<span class='danger'>[H] recoils and stumbles from the attack!</span>", \
+									"<span class='userdanger'>[H] recoils and stumbles from the attack!</span>")
+					H.apply_effect(0.5, WEAKEN, armor_block)
```

Comment:
0.5 weaken effect is the same as 1 weaken, since I'm 100% sure all of the effects are tick-based instead of delay based.